File: app/src/main/java/com/example/ruslanyussupov/popularmovies/ItemDecoration.java
package com.example.ruslanyussupov.popularmovies;


import android.graphics.Rect;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class ItemDecoration extends RecyclerView.ItemDecoration {

    private final int offsetLeft;
    private final int offsetTop;
    private final int offsetRight;
    private final int offsetBottom;

    public ItemDecoration(int offsetLeft, int offsetTop, int offsetRight, int offsetBottom) {
        this.offsetLeft = offsetLeft;
        this.offsetTop = offsetTop;
        this.offsetRight = offsetRight;
        this.offsetBottom = offsetBottom;
    }

    @Override
    public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
        super.getItemOffsets(outRect, view, parent, state);
        outRect.set(offsetLeft, offsetTop, offsetRight, offsetBottom);
    }

}

File: app/src/main/java/com/example/ruslanyussupov/popularmovies/ItemDecoration.kt
package com.example.ruslanyussupov.popularmovies


import android.graphics.Rect
import androidx.recyclerview.widget.RecyclerView
import android.view.View

class ItemDecoration(private val offsetLeft: Int,
                     private val offsetTop: Int,
                     private val offsetRight: Int,
                     private val offsetBottom: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        super.getItemOffsets(outRect, view, parent, state)
        outRect.set(offsetLeft, offsetTop, offsetRight, offsetBottom)
    }

}
